Add user stories to README

ChooJiaXin opened the pull request:
closes #46

ChooJiaXin committed:
Add user stories to README

andreatanky reviewed (commented):
I have changed some of these user stories to make it more specific. eg. 'work'
Perhaps refer to DG to see which is better?

ChooJiaXin committed:
Update user stories

ChooJiaXin commented:
> I have changed some of these user stories to make it more specific. eg. 'work'
> Perhaps refer to DG to see which is better?

Updated accordingly :D

andreatanky commented:
LGTM!

itsjerryho reviewed (commented):
Looks good! Just a small suggestion, maybe we can standardise the use of capital letters under 'I can' and 'so that I can'. we can either capitalise the first word or not capitalise the first word for both scenarios :-)

ChooJiaXin commented:
> Looks good! Just a small suggestion, maybe we can standardise the use of capital letters under 'I can' and 'so that I can'. we can either capitalise the first word or not capitalise the first word for both scenarios :-)

Sounds good! Which do you suggest is better? I'm fine with either way :)

itsjerryho commented:
> > Looks good! Just a small suggestion, maybe we can standardise the use of capital letters under 'I can' and 'so that I can'. we can either capitalise the first word or not capitalise the first word for both scenarios :-)
> 
> Sounds good! Which do you suggest is better? I'm fine with either way :)

I think for now we can not capitalise for both scenarios !

ChooJiaXin committed:
Update README according to review comments

ChooJiaXin commented:
> > > Looks good! Just a small suggestion, maybe we can standardise the use of capital letters under 'I can' and 'so that I can'. we can either capitalise the first word or not capitalise the first word for both scenarios :-)
> > 
> > 
> > Sounds good! Which do you suggest is better? I'm fine with either way :)
> 
> I think for now we can not capitalise for both scenarios !

Amended :)
